src/test/fixtures/blueprints.ts

```diff
@@ -322,7 +322,7 @@ export const mockCentosBlueprintComposes: GetBlueprintComposesApiResponse = {
       created_at: '2021-04-27T12:31:12Z',
       blueprint_version: 1,
       request: {
-        distribution: 'centos-8',
+        distribution: 'rhel-10-nightly',
```

Comment:
Is this why you needed to delete the test?
I think we could convince the handler to return special case centos-8 (copy paste it into the centos-8 test) in the test itself. MSW can update single mock in test itself